site/docs/remote-caching.md

```diff
@@ -133,16 +133,21 @@ location /cache/ {
 }
 ```
 
-### Bazel Remote Cache
+### bazel-remote
 
-Bazel Remote Cache is an open source remote build cache that you can use on
-your infrastructure. It is experimental and unsupported.
+bazel-remote is an open source remote build cache that you can use on
+your infrastructure. It is experimental, but has been successfully used in
+production at several companies since early 2018. Note that the Bazel project
+does not provide technical support for bazel-remote.
```

Comment:
It has successfully been used in production at several companies since early 2018. Note that the Bazel project
does not provide technical support for bazel-remote.

Reply:
Updated.